pilot/pkg/features/pilot.go

```diff
@@ -105,6 +105,12 @@ var (
 			" EDS pushes may be delayed, but there will be fewer pushes. By default this is enabled",
 	).Get()
 
+	SendUnhealthyEndpoints = env.RegisterBoolVar(
+		"PILOT_SEND_UNHEALTHY_ENDPOINTS",
+		true,
```

Comment:
Suggest false by default, this may have performance influence

Reply:
The reason I set default to true is this is needed for features like warmup etc which we are going to build. When that is used we can not ask customers to set this. If they are not using those features and they see it as perf issue, they can disable it.